mono/utils/mono-os-mutex.h

```diff
@@ -118,12 +118,22 @@ mono_os_mutex_unlock (mono_mutex_t *mutex)
 		g_error ("%s: pthread_mutex_unlock failed with \"%s\" (%d)", __func__, g_strerror (res), res);
 }
 
+#if !defined(PLATFORM_MACOSX)	
+	pthread_condattr_t attr;
```

Comment:
w32handle.c has used static mono_cond_t so , I had applied the same logic here. but now it is local